util/test/chpl_launchcmd.py

```diff
@@ -143,6 +145,10 @@ def full_test_command(self, output_file, error_file):
                 self.test_command[0]))
             full_test_command += ['test', '-f', self.test_command[0], '&&']
 
+        envvars = ['LANG', 'LC_ALL',  'LC_COLLATE']
+        charset = ' '.join(['{0}="{1}"'.format(e, os.getenv(e, '')) for e in envvars])
```

Comment:
I don't know if it's a problem but I worry about the case that these variables aren't set in the caller. Would say `export LANG=""` will cause different behavior compared with not having an `export LANG` at all. Would it make sense here to only add the environment variable settings when they are set?

Reply:
We unset `LC_ALL` in start_test, so here I want to export the fact that we've unset it. Similar to

https://github.com/chapel-lang/chapel/blob/bbdc16d0574f0d779a5a6ca49767f12593afce19/runtime/src/chpl-launcher-common.c#L548-L551